src/Symfony/Bridge/Doctrine/Tests/DoctrineTestHelper.php

```diff
@@ -61,7 +61,10 @@ public static function createTestConfiguration(): Configuration
         if (class_exists(DefaultSchemaManagerFactory::class)) {
             $config->setSchemaManagerFactory(new DefaultSchemaManagerFactory());
         }
-        $config->setLazyGhostObjectEnabled(true);
+
+        if (\version_compare(\Composer\InstalledVersions::getVersion('doctrine/persistence') ?? '999.999.999', '3.1.0', '>=')) {
```

Comment:
I'd rather go with a feature test. Checkes using InstalledVersions should be avoided to me, reserved for special cases.

Reply:
such a feature test could be checking for the `Doctrine\Persistence\Mapping\Driver\AnnotationDriver` to exist (it was removed in `doctrine/persistence` 3.0)

Reply:
Thx for the quick review. Update the code to use class_exists now.